ghga_datasteward_kit/s3_upload/config.py

```diff
@@ -40,22 +40,41 @@ class LegacyConfig(BaseSettings):
     Required options for legacy file uploads.
     """
 
-    s3_endpoint_url: SecretStr = Field(..., description="URL of the S3 server")
+    s3_endpoint_url: SecretStr = Field(
+        ..., description="URL of the local data hub's S3 server."
+    )
     s3_access_key_id: SecretStr = Field(
-        ..., description="Access key ID for the S3 server"
+        ...,
+        description=(
+            "Access key ID for the local data hub's S3 server. The corresponding"
+            + " account need to have s3:GetObject and s3:PutObject privileges for the"
+            + " internal staging bucket (as in `bucket_id`). These credentials should"
+            + " never be shared with GHGA Central."
+        ),
     )
     s3_secret_access_key: SecretStr = Field(
-        ..., description="Secret access key for the S3 server"
+        ...,
+        description=("Secret access key corresponding to the `s3_access_key_id`."),
```

Comment:
This one is still a bit noninformative and might be confusing. Probably better to clarify that there's always a pair of `access_key_id` and `secret_access_key` needed to access S3, then this makes more sense to someone never confronted with the concept

Reply:
Might be part of the long description of the access key ID, then this can be left as is.

Reply:
Good suggestion.

Reply:
Done